configs/node.js

```diff
@@ -0,0 +1,31 @@
+import nodePlugin from 'eslint-plugin-n';
+import jsonFiles from 'eslint-plugin-json-files';
+import globals from 'globals';
+import javascript from './javascript.js';
+
+export default [
+  nodePlugin.configs['flat/recommended'],
+  ...javascript,
+  {
+    files: ['**/*.json'],
```

Comment:
wondering if we should break these out into a separate `json` rulset, and if so, if we should include it in this ruleset definition or leave it separate for developers to mix in as they choose?